octobot_trading/exchanges/exchange_manager.py

```diff
@@ -16,6 +16,8 @@
 import uuid
 from copy import copy
 
+from octobot_backtesting.api.backtesting import start_backtesting
```

Comment:
I don't think the backtesting API should be call in ExchangeManager. I think ExchangeSimulator is the right class to call Backtesting APIs.